packages/components/src/components/pagination/pagination.tsx

```diff
@@ -87,7 +87,8 @@ export class Pagination {
   @Watch('totalElements')
   calculateWidth() {
     // calculate max possible width
-    this.maxWidth = (this.totalElements.toString().length * 3 + 3) * 9;
+    this.maxWidth = (this.totalElements.toString().length * 4 + 3) * 9;
```

Comment:
I know this is not yours, but these "magic" numbers are a bit intriguing, do you know whey they are like that?

Reply:
I was wondering if something like `white-space: nowrap;` in CSS would help?

Reply:
I don't know how it came to this formula, but it seems to work brilliantly :-)
It is a good idea to give the info part, the one with the numbers a fixed width, because every button click could change this. I went back to the original formula and used white-space: nowrap; Thank you for the hint!